src/test/java/org/sagebionetworks/bridge/sdk/integration/StudyTest.java

```
@@ -80,9 +82,38 @@ public void crudStudy() throws Exception {
         }
     }
 
-    // Testing a researcher requires that the host be hacked so that the researcher matches
-    // the host name of this newly created study. That's beyond the debugging setup of the 
-    // server and the SDK at the moment. Would need to set a header, I think, with the host
-    // that came back in the study.
+    @Test
+    public void researcherCannotAccessAnotherStudy() {
+        study = new Study();
+        study.setIdentifier(Tests.randomIdentifier());
+        study.setMinAgeOfConsent(18);
+        study.setMaxNumOfParticipants(100);
+        study.setName("Test Study [SDK]");
+        study.setTrackers(Lists.newArrayList("sage:A", "sage:B"));
+        
+        AdminClient client = admin.getSession().getAdminClient();        
+        client.createStudy(study);
+        
+        TestUser researcher = TestUserHelper.createAndSignInUser(StudyTest.class, false, Tests.TEST_KEY+"_researcher");
+        try {
+            researcher.getSession().getAdminClient().getStudy(study.getIdentifier());
+            fail("Should not have been able to get this other study");
+        } catch(UnauthorizedException e) {
+            assertEquals("Unauthorized HTTP response code", 403, e.getStatusCode());
+        } finally {
+            researcher.signOutAndDeleteUser();
+        }
+        
+    }
+    
+    @Test
+    public void researcherCanAccessStudy() {
+        TestUser researcher = TestUserHelper.createAndSignInUser(StudyTest.class, false, Tests.TEST_KEY+"_researcher");
+        
+        ResearcherClient rclient = researcher.getSession().getResearcherClient();
```

So, if you're _not_ a researcher, then this method should throw an exception.